.travis.yml

```diff
@@ -137,7 +137,71 @@ jobs:
         # create the github release
         - release_github_v2.sh "$TRAVIS_TAG"
         # attach generate_secret to the github release
-        - $TRAVIS_BUILD_DIR/scripts/ci_build_generate_secret.sh && hub release edit "$TRAVIS_TAG" -m "" -a "/tmp/output_packages/generate_secret#generate_secret for Linux 64-bit"
+        - $TRAVIS_BUILD_DIR/scripts/ci_build_generate_secret.sh && $TRAVIS_BUILD_DIR/scripts/ci_attach_generate_secret.sh
+
```

Comment:
i have to attach 3 tarballs, making this line super long so `ci_attach_generate_secret.sh` was created instead so i can use a for loop and keep things tidy